### PR TITLE
Mannings2020

### PR DESCRIPTION
**Pull request by almannin**

### CODE REVIEW

**Review by SunilSimha — CHANGES_REQUESTED** [review]
Looks good to me. I've requested a few docstrings to be completed but the rest of it looks fine. Is there a test for the fractional flux code?

File: frb/data/Galaxies/121102/FRB121102_host.json [hunk 1]
```diff
@@ -47,12 +53,12 @@
...
-        "ang_avg_err": 0.0027529576372375447,
-        "ang_best": 0.12450060896549818,
-        "ang_best_err": 0.002753795763727014,
-        "physical": 0.41179423877492893,
-        "physical_err": 0.009108366936420729
+        "ang_avg": 0.22738211697817798,
+        "ang_avg_err": 0.015495714927742417,
+        "ang_best": 0.2268624805850244,
+        "ang_best_err": 0.015504425274835199,
+        "physical": 0.7503630968181939,
```
Comment (keheintz) [inline frb/data/Galaxies/121102/FRB121102_host.json:60]: Looks too high, Bassa+17 estimate physical offset = 0.18 kpc (and it's even higher than our own initial analysis
)

**Review by keheintz — APPROVED** [review]

**Comment by almannin** [comment]
> Looks good to me. I've requested a few docstrings to be completed but the rest of it looks fine. Is there a test for the fractional flux code?

Okay I think I have addressed these!

**Review by SunilSimha — APPROVED** [review]

File: frb/data/Galaxies/121102/FRB121102_host.json (hunk 1 above)
Comment (profxj) [reply frb/data/Galaxies/121102/FRB121102_host.json:60]: That offset is from the peak of the SF not the center of the galaxy.

**Comment by profxj** [comment]
Tests passing on my machine:

================= 52 passed, 2 skipped, 36 warnings in 41.89s ==================

**Comment by profxj** [comment]
merging despite CI failures.  Hopefully NOAO lab will work again one of these days..

**Pull request merged**